kaocha-runner.el

```diff
@@ -210,7 +231,7 @@ If BACKGROUND? is t, we don't message when the tests start running."
            (when (and (< 1 (- (float-time) start-time))
                       (not shown-details?))
              (setq shown-details? t)
-             (kaocha-runner--show-details-window original-buffer 12)))
+             (kaocha-runner--show-details-window original-buffer kaocha-runner-tests-win-min-height)))
```

Comment:
How about `kaocha-runner-ongoing-tests-win-min-height`? It's a mouthful, but it's a bit weird too. I think the docstring should try to explain when it is used: the height of the output window when tests are taking long to run, to show the ongoing progress from kaocha.

Reply:
Done